Dockerfile

```diff
@@ -1,13 +1,20 @@
-FROM busybox:latest
-MAINTAINER Matt Kemp <matt@mattikus.com>
+FROM busybox:latest AS builder
+LABEL MAINTAINER="Matt Kemp <matt@mattikus.com>"
 
 ENV version=1.3.2
```

Comment:
Version is currently 1.3.3. Lets keep parity with HEAD.

Reply:
:+1: